LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2008, Batiste Bieler
+Copyright (c) 2008-2018, Batiste Bieler
```

Comment:
Please use 2008-present to avoid the update every year.

Reply:
Updated